### PR TITLE
[VA Forms] Create template for Form Detail pages

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
This PR implements the template for the new Form Detail pages into the preview server.

Mockup - https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/find-va-forms-redesign/form-detail-landing-pages/WIP-design-files/Generic-form-page-with-May-2020-updates.pdf

I still need to implement the Related Links section. I don't think the content model supports it right now.

https://github.com/department-of-veterans-affairs/va.gov-team/issues/10606

## Testing done
Tested against several VA forms using the preview server locally. 

### http://localhost:3001/preview?nodeId=5776
image

## Acceptance criteria
- [ ] Form Detail page template has been started

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/stages/build/drupal/graphql/vaForms.graphql.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  }
+  fieldVaFormIssueDate {
+    value
+    date
+  }
+  fieldVaFormNumPages
+  status
+  moderationState
+}
+`;
```
Comment (ncksllvn) [inline src/site/stages/build/drupal/graphql/vaForms.graphql.js:45]: Copied from https://dsva.slack.com/archives/CUB5X5MGF/p1592940541321700 and tested on https://prod.cms.va.gov/graphql/explorer

**Review by kelsonic — APPROVED** [review]
Wow! Awesome!! 🔥 😎

File: src/site/layouts/va_form.drupal.liquid [hunk 2]
```diff
@@ -0,0 +1,70 @@
+{% include "src/site/includes/header.html" with drupalTags = true %}
+{% include "src/site/includes/alerts.drupal.liquid" %}
+{% include "src/site/includes/preview-edit.drupal.liquid" %}
+{% include "src/site/includes/breadcrumbs.drupal.liquid" %}
+<div id="content" class="interior">
+  <main class="va-l-detail-page">
```
Comment (kelsonic) [inline src/site/layouts/va_form.drupal.liquid:6]: Are we sure we don't have a `main` tag as a parent?

Comment (ncksllvn) [reply src/site/layouts/va_form.drupal.liquid:6]: I don't think so. I followed the `page.drupal.liquid` template as an example

**Review by rianfowler — APPROVED** [review]

**Pull request merged**